client/single_components/form_fields/drop_down.jsx

```diff
@@ -62,22 +65,27 @@ class DropDown extends React.Component {
 
   render() {
     const { items, dropDownStyle, tooltip } = this.props;
+    const { openMenu } = this.state;
+    const selectedValue = this.state.value;
```

Comment:
lets select it here:   `const { isMenuOpen, value: selectedValue } = this.state;`